### PR TITLE
`Slick.Plugins.Resizer` now has `setOptions()`

### PR DESCRIPTION
**Pull request by arashdalir**

### CODE REVIEW

File: plugins/slick.resizer.js [hunk 1]
```diff
@@ -85,8 +85,11 @@
       rightPadding: 0
     };
 
+    function setOptions(_newOptions){
+      options = $.extend(true, {}, _defaults, _newOptions);
+    }
     function init(grid) {
```
Comment (ghiscoding) [inline plugins/slick.resizer.js:91]: nitpick can you add empty line between the 2 functions for a better separation

File: plugins/slick.resizer.js [hunk 2]
```diff
@@ -119,14 +122,17 @@
...
         $(window).on('resize.grid.' + _gridUid, function (event) {
-          _self.onGridBeforeResize.notify({ grid: _grid }, event, _self);
-
-          // unless the resizer is paused, let's go and resize the grid
-          if (!_resizePaused) {
-            // for some yet unknown reason, calling the resize twice removes any stuttering/flickering
-            // when changing the height and makes it much smoother experience
-            resizeGrid(0, newSizes, event);
-            resizeGrid(0, newSizes, event);
+          if (event.originalEvent.type !== 'resize')
```
Comment (ghiscoding) [inline plugins/slick.resizer.js:125]: another nitpick can you move the opening curly brace on the same line as the `if`. Also I'm curious to know, what does this line help with? What other event type can show up here?

**Review by ghiscoding — CHANGES_REQUESTED** [review]

File: plugins/slick.resizer.js (hunk 2 above)
Comment (arashdalir) [reply plugins/slick.resizer.js]: oopsie! sorry! this was a try to break an endless loop, should not have been committed. removed it in the new push.

File: plugins/slick.resizer.js (hunk 1 above)
Comment (arashdalir) [reply plugins/slick.resizer.js:92]: think now it should look better

**Pull request merged**